### ISSUE TITLE
I am going to upgrade something in new repo, and fix some bugs when using webpack

### ISSUE THREAD

**Issue by hxl-dy** [flavioribeiro/clappr-rtmp-plugin#82]
repo: https://github.com/hxl-dy/clappr-rtmp-plugin/tree/upgrade

upgrades: 
1. jshint -> eslint
2. webpack1 -> webpack2

fix bugs:
1. cannot auto play
2. play button won't change to pause when playing and no action occurs after click it
3. double click video screen won't trigger full screen action
4. volume control doesn't work

**Comment by flavioribeiro** [comment]
yep @hxl-dy feel free to reach me any time you get stuck, thanks for helping on the maintenance of this project

**Comment by hxl-dy** [comment]
repo: https://github.com/hxl-dy/clappr-rtmp-plugin/tree/webpack2

DONE:
1. jshint -> eslint
2. webpack1 -> webpack2

TODO: 
1. use uglify#harmony
2. remove babel

FIXME:
1. double click video screen won't trigger full screen action

**Comment by flavioribeiro** [comment]
@hxl-dy I wonder if it makes sense to bring the modifications in your branch to this repository, are you still actively using this plugin?
